src/components/Chip/Chip.scss

```diff
@@ -0,0 +1,6 @@
+.Chip {
+    font-size: var(--font-size-small);
```

Comment:
Let's increase the size of `--font-size-small`, it's current size is hardly readable. 12 or 13 pixels would be better.